.gitignore

```diff
@@ -112,7 +112,7 @@ compilation-stats.json
 /packages/grafana-e2e/cypress/logs
 /e2e/server.log
 /e2e/suite1/screenshots
-!/e2e/suite1/screenshots/expeced/*
+!/e2e/suite1/screenshots/expected/*
```

Comment:
This could easily have an impact on things. It's also almost certainly a bug-fix